charts/incubator/tt-rss/values.yaml

```diff
@@ -0,0 +1,242 @@
+image:
+  repository: cthulhoo/ttrss-fpm-pgsql-static
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
+nginxImage:
+  repository: cthulhoo/ttrss-web-nginx
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f963b38ef8695d05c7f219e9405648dcb2f1f3c0b81609353bb1ba078bc8e318
+updaterImage:
+  repository: cthulhoo/ttrss-fpm-pgsql-static
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
```

Comment:
lets change to latest, and let bot update the digest when available.
I don't think it would be able to figure out hashes as tags